### ISSUE TITLE
mangaNT new module missing/unavailable

### ISSUE THREAD

**Issue by tachikoma-days** [fmd-project-team/FMD#777]
MangaNT db file download fails. couldn't find it here anywhere to download.

**Comment by Slasar41** [comment]
@Tmp341 Could you add it to database repository?

**Comment by Tmp341** [comment]
Website added 17 days ago, and nobody added it's database to the server.

**Comment by Tmp341** [comment]
Updating now. Most likely upload it tomorrow.

**Comment by Tmp341** [comment]
@Slasar41 Other website databases, when i update and stop one halfway, it normally shows how many mangas (for example 100) it got. But MangaNT update just becomes empty. I accidently closed my pc last night, and lost 26k manga out of 29k. Now i need to start again.

**Comment by Slasar41** [comment]
Yeah sometimes when I stopped updating halfway it'll become empty, it's happened with other websites too.
I don't know why, you can open the log and see the error information.
